src/uct/tcp/tcp_iface.c

```diff
@@ -29,8 +29,8 @@ static ucs_config_field_t uct_tcp_iface_config_table[] = {
   {"TX_SEG_SIZE", "8kb",
    "Size of send copy-out buffer",
    ucs_offsetof(uct_tcp_iface_config_t, tx_seg_size), UCS_CONFIG_TYPE_MEMUNITS},
-  
-  {"RX_SEG_SIZE", "64kb",
+
+  {"RX_SEG_SIZE", "8kb",
```

Comment:
we used 64kb for AM/Zcopy

Reply:
not sure i understand the point? the issue is on recv side. if we use 64k am_zcopy with multiple frags it's ok. problem is in mdesc pool on the worker - which is used for unexpected messages - even for an inline receive of as little as 64b, it would consume 64k.

Reply:
yes, I see.
I meant that we use this 64kb buffer to send-receive large AM/Zcopy as a single `uct_ep_am_zcopy()` transaction

Reply:
i see, so i guess this change may have some effect on TCP performance,, but IMO better reduce memory usage
in general maybe we need to support tx_seg>rx_seg in UCT API

Reply:
> maybe we need to support tx_seg>rx_seg in UCT API

yes, I think so